include/mbedtls/config.h

```diff
@@ -2333,6 +2333,17 @@
  */
 #define MBEDTLS_SHA512_C
 
+/**
+ * \def MBEDTLS_SSL_ASYNC_PRIVATE_C
+ *
+ * Enable asynchronous external private key operations in SSL. This allows
+ * you to configure an SSL connection to call an external cryptographic
+ * module to perform private key operations instead of performing the
+ * operation inside the library.
+ *
+ */
+#define MBEDTLS_SSL_ASYNC_PRIVATE_C
```

Comment:
The name (and place in `config.h`) suggests this is going to be implemented as a full-blown module with its own `.c` file, which is not the case. Please rename and place together with the other SSL options.